doc/conf.py

```diff
@@ -337,7 +337,7 @@
 		('boards/arm/msp_exp432p401r_launchxl/doc/msp_exp432p401r_launchxl', 'boards/arm/msp_exp432p401r_launchxl/doc/index'),
 		('boards/arm/nrf51_ble400/doc/nrf51_ble400', 'boards/arm/nrf51_ble400/doc/index'),
 		('boards/arm/nrf51_blenano/doc/nrf51_blenano', 'boards/arm/nrf51_blenano/doc/index'),
-		('boards/arm/nrf51_pca10028/doc/nrf51_pca10028', 'boards/arm/nrf51_pca10028/doc/index'),
+		('boards/arm/nrf51dk_nrf51422/doc/nrf51dk_nrf51422', 'boards/arm/nrf51dk_nrf51422/doc/index'),
```

Comment:
this looks wrong, it will not create a redirect from old page to the new one.

Reply:
You need two entries:

```
('boards/arm/nrf51_pca10028/doc/nrf51_pca10028', 'boards/arm/nrf51dk_nrf51422/doc/index'),
('boards/arm/nrf51_pca10028/doc/nrf51_pca10028/index', 'boards/arm/nrf51dk_nrf51422/doc/index')

```

I think...

Reply:
updated, thanks